inst/plumber/unbiased_api/minimisation_pocock.R

```diff
@@ -212,18 +212,19 @@ function(identifier, name, method, arms, covariates, p, req, res) {
 #*
 
 function(study_id, current_state, req, res) {
+  collection <- checkmate::makeAssertCollection()
   # Assertion connection with DB
-  checkmate::assert(DBI::dbIsValid(db_connection_pool), .var.name = "DB connection")
```

Comment:
This assert is not needed here. We are using connection pool, so connection will always be "fresh" and valid.

Reply:
Even if someone forgets to give the credentials?

Reply:
Yes, in that case connection pool creation will fail and app won't start at all